src/app.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import config from "./config";
+import router from "./routes/";
+
+// Initialize express
+const app = express();
+
+// Middleware to parse JSON bodies
+app.use(express.json());
+
+// Use the router
+app.use(router);
+
+app.get("/", (req, res) => {
+  res.json({
+    message: "Todo App",
+  });
+});
+
```

Comment:
app.get(errorHandler) is the last piece of middleware =>  Parses App failures in response json format

{success: false,
message: "...",
statusCode: 500
}